packages/renderer/src/lib/preferences/PreferencesConnectionActions.svelte

```diff
@@ -144,7 +144,7 @@ function getLoggerHandler(
   {#if connection.lifecycleMethods && connection.lifecycleMethods.length > 0}
     <div class="mt-2 relative">
       <!-- TODO: see action available like machine infos -->
-      <div class="flex bg-charcoal-800 w-fit rounded-lg m-auto">
+      <div class="flex bg-charcoal-800 w-fit rounded-lg m-auto" role="group" aria-label="ConnectionActions">
```

Comment:
Just wondering is the role attribute required ? Shouldn't the label be enough ?

Reply:
It's not strictly required but it allows for more precise locating in playwright in testing, since getByRole()  has more options to work with than getByLabel().